script/tool/lib/src/version_check_command.dart

```diff
@@ -655,4 +690,55 @@ ${indentation}The first version listed in CHANGELOG.md is $fromChangeLog.
 
     return null;
   }
+
+  Future<bool> _validateBatchRelease({
```

Comment:
`_validatePendingChangeForBatchReleasePackage`? The state this method is validating is explicitly *not* a batch release, as the current name might suggest to a reader, but the non-release commit for a package using batched releases.